test/integration/sync-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+buster.testCase('=> Sync', {
+  prepare: function() {
+    this.timeout = 90e3;
+  },
+
+  'can be read': function() {
```

Comment:
Yeah, I thought about a way of testing the next sync but there's no good way of doing that.